src/ExpenseTrackerGrupo2/Persistence/Database/Scripts/init.sql

```diff
@@ -0,0 +1,87 @@
+-- User
```

Comment:
Delete the comments to make the file cleaner

Reply:
Done.